src/qml/NightstandPage.qml

```diff
@@ -0,0 +1,194 @@
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    height: Dims.h(20)
+                }
+
+                IconButton {
+                    width: Dims.w(20)
+                    height: width
+                    iconName: "ios-add-circle-outline"
+                    edge: undefinedEdge
+                    anchors.right: parent.right
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    onClicked: {
+                        var newVal = nightstandBrightness.value + 10
+                        if(newVal > 100) newVal = 100
+                        nightstandBrightness.value = newVal
+                    }
+                }
+            }
+
+            Item {
+                id: delaySetting
+                height: Dims.h(30)
+                Layout.fillWidth: true
+                Layout.columnSpan: 2
+                Label {
+                    //% "Nightstand delay"
+                    text: qsTrId("id-nightstand-delay")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+
+                IconButton {
+                    iconName: "ios-remove-circle-outline"
+                    edge: undefinedEdge
+                    anchors.left: parent.left
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    onClicked: {
+                        var newVal = nightstandDelay.value - 5
+                        if(newVal < 0) newVal = 0
+                        nightstandDelay.value = newVal
+                    }
+                }
+
+                Label {
+                    text: nightstandDelay.value + "s"
+                    font.pixelSize: Dims.l(6)
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    height: Dims.h(20)
+                }
+
+                IconButton {
+                    width: Dims.w(20)
+                    height: width
+                    iconName: "ios-add-circle-outline"
+                    edge: undefinedEdge
+                    anchors.right: parent.right
+                    anchors.top: parent.top
+                    anchors.topMargin: Dims.h(10)
+                    onClicked: {
+                        var newVal = nightstandDelay.value + 5
+                        if(newVal > 30) newVal = 30
+                        nightstandDelay.value = newVal
+                    }
+                }
+            }
+
+            Label {
+                //% "Nightstand enable"
```

Comment:
Naturally I'd say this should come first.

And it would be awesome if when Nighstand is disabled the other option (brightness) would be greyed out and unusable ;)

Reply:
Both done!